python/GafferSceneUI/LightTweaksUI.py

```diff
@@ -178,6 +178,33 @@ def _updateFromPlug( self ) :
 		for i in ( 0, 2, 3 ) :
 			self.__row[i].setEnabled( enabled )
 
+
+def __deletePlug( plug ) :
+
+	with Gaffer.UndoScope( plug.ancestor( Gaffer.ScriptNode ) ) :
+		print plug, plug.parent()
+		plug.parent().removeChild( plug )
+
+def __plugPopupMenu( menuDefinition, plugValueWidget ):
+
+	plug = plugValueWidget.getPlug()
+	parent = plug.parent()
+	node = plug.node()
+
+	if not isinstance( node, GafferScene.LightTweaks ) or not parent.parent().isSame( node["tweaks"] ) :
+		return
+
+	menuDefinition.append( "/DeleteDivider", { "divider" : True } )
+	menuDefinition.append(
+		"/Delete Tweak",
```

Comment:
Yeah, I think you're right that just "Delete" would be better, so it feels like the same menu item as the regular one.

Reply:
One thought I had that I didn't really follow up on was that potentially we also have a 'Delete' entry for widgets that manipulate strings? The usual copy/paste/cut/delete combo? Not sure if that's ever the case anywhere in the code, but if it is we might want to consider a different name for deleting elements from the UI? Can you think of anything that would clash?